obip-peer-endorsements.md

```diff
@@ -0,0 +1,159 @@
+<pre>
+OBIP: X
+Title: 3rd Party Peer Endorsements
+Author: Tyler Smith <tyler@ob1.io>
+Discussions-To: Github Issues
+Status: Draft
+Type: Standards Track
+Created: 02/28/2018
+Copyright: MIT
+Replaces: 8
+</pre>
+
+## Abstract
+A change to the OBIP8 Verified Moderators system to make it a more generic and flexible service. It outlines a specification for a system allowing 3rd parties to provide a list of peers that it is endorsing in some way, with the nature of the endorsements being defined by the 3rd party itself.
+
+## Motivation
+Allow users on OpenBazaar to have more information about the identity and reputation of peers that they interact with. If a 3rd party can provide accurate endorsements then users can make safer decisions.
+
+## Endorsement endpoint
+
+To provide endorsements a provider must have a public endpoint returning data with the following schema:
+
+#### Example
+
+```json
+{
+  "data": {
+    "name": "BazaarCo",
+    "description": "Peers Endorsed by BazaarCo",
+    "link": "https://bazaarco.example.com/endorsements"
+  },
+  "types": [
+    {
+      "name": "standard",
+      "description": "A peer that has been vetted by BazaarCo",
+      "badge": "https://bazaarco.example.com/standard_badge.png"
```

Comment:
Since the types are so different from each other, should they each have a link? I can imagine some providers would want a page, or a section of a page, for each one.